### PR TITLE
[Merged by Bors] - chore: tidy various files

### PR DESCRIPTION
**Pull request by Ruben-VandeVelde**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Algebra/Hom/Centroid.lean [hunk 1]
```diff
@@ -112,6 +111,7 @@ instance : CoeFun (CentroidHom α) fun _ ↦ α → α :=
 /- Porting note:
 `theorem to_fun_eq_coe {f : CentroidHom α} : f.toFun = (f : α → α) := rfl`
 removed because it is now a tautology -/
+#noalign centroid_hom.to_fun_eq_coe
```
Comment (urkud) [inline Mathlib/Algebra/Hom/Centroid.lean:114]: Is this still true? Don't we use `FunLike.coe` on the RHS?

Comment (Ruben-VandeVelde) [reply Mathlib/Algebra/Hom/Centroid.lean]: Nice catch, thanks

File: Mathlib/Algebra/Hom/Centroid.lean [hunk 2]
```diff
@@ -498,7 +492,7 @@ theorem toEnd_int_cast (z : ℤ) : (z : CentroidHom α).toEnd = ↑z :=
   rfl
 #align centroid_hom.to_End_int_cast CentroidHom.toEnd_int_cast
 
-instance : Ring (CentroidHom α) :=
+instance ring : Ring (CentroidHom α) :=
```
Comment (riccardobrasca) [inline Mathlib/Algebra/Hom/Centroid.lean:495]: Shouldn't this be `instRing` or something like that?

**Review by riccardobrasca — COMMENTED** [review]
Thanks!

bors d+

File: Mathlib/Algebra/Hom/Centroid.lean (hunk 2 above)
Comment (Ruben-VandeVelde) [reply Mathlib/Algebra/Hom/Centroid.lean]: Oh, I was being consistent with `CentroidHom.commRing`, but that's not an instance. Thanks for pointing that out.

**Pull request closed without merging**